bands/configspec.ini

```diff
@@ -27,12 +29,11 @@ bands = string(default="Band Management")
 [[band_merch]]
 no_merch = string(default="Not selling merch")
 own_table = string(default="Dedicated table")
-rock_island = string(default="Rock Island")
```

Comment:
this is the only questionable thing in this PR, having to remove this line.

My goal is to check if c.ROCK_ISLAND exists and then do different things in the template.

If configspec.ini has 'rock_island' in this enum, but the development.ini does NOT have it, c.ROCK_ISLAND will still created anyway.  There doesn't seem to be a way to remove/override part of an enum defined in configspec.ini. 

So the only way I could make this work was to remove rock_island here, and have prime and labs add it back in in their .ini files. (magstock does not add it back in)